packages/core/src/__tests__/lint.test.ts

```diff
@@ -105,6 +105,40 @@ describe('lint', () => {
     `);
   });
 
+  it("'plugins' shouldn't be allowed in 'apis' -> 'lint' field", async () => {
+    const document = parseYamlToDocument(
+      outdent`
+      apis: 
+        lint: 
+          plugins: 
+            - './local-plugin.js'
+      lint:
+        plugins:
+          - './local-plugin.js'
+      `,
+      '',
+    );
+    const results = await lintConfig({ document });
+
+    expect(replaceSourceWithRef(results)).toMatchInlineSnapshot(`
+      Array [
+        Object {
+          "location": Array [
+            Object {
+              "pointer": "#/apis/lint/plugins",
+              "reportOnKey": true,
+              "source": "",
+            },
+          ],
+          "message": "Property \`plugins\` is not expected here.",
```

Comment:
PR looks good to me. Are we sure this is actually correct though? I heard:

- apis lint rules (`preprocessors`, `rules`, `decorators`) merge and take priority over the global configuration
- other things override the global configuration (this was explicitly `extends`, but I'm unclear is `plugins` explicitly not allowed)?

Reply:
Yes. As far as I remember, we discussed `plugins` usage with @RomanHotsiy and decided it should be declared only on the top level.

Reply:
We'll adjust the docs PR.